tasks/main.yml

```diff
@@ -20,6 +20,16 @@
     - 4063
     - 4064
 
+- name: OMERO external load balanced security group rules
+  os_security_group_rule:
+    direction: ingress
+    port_range_max: 14079
```

Comment:
Would we want to make this range configurable?

Reply:
In theory yes. In practice this role has a single use, and the rules are shared across all IDR deployments, so I think it's safest not to make this configurable.